src/python/pants/backend/python/target_types_rules.py

```diff
@@ -489,7 +500,13 @@ async def validate_python_dependencies(
     request: PythonValidateDependenciesRequest,
     python_setup: PythonSetup,
 ) -> ValidatedDependencies:
-    dependencies = await MultiGet(Get(WrappedTarget, Address, d) for d in request.dependencies)
+    dependencies = await MultiGet(
+        Get(
+            WrappedTarget,
+            WrappedTargetRequest(d, description_of_origin=f"the dependencies of {request.address}"),
```

Comment:
@stuhood is this right? It's the dependencies validation plugin hook you added.

Reply:
Correct.

Reply:
Cool, thanks. Also I think this is unlikely to actually trigger - seems like we will have errored earlier on when computing the dependencies in the first place.